packages/http/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@stoplight/prism-core": "^3.1.1",
+    "@stoplight/ref-resolvers": "^1.0.2",
```

Comment:
Hmm, the json-ref-resolver should probably be next to the ref-resolvers package; if not, something is wrong. Can you check it out?

Reply:
Good spot. I moved `json-ref-resolver` to http package where it is actually used.

Reply:
Oh, the same applies to yaml package.

Reply:
Fix them all 🚎